webpack.config.js

```diff
@@ -7,34 +7,26 @@ const webpack = require('webpack');
 const license = fs.readFileSync(path.join(__dirname, 'LICENSE')).toString();
 
 module.exports = {
-  entry: [
-    'babel-polyfill',
-    './jstp',
-  ],
+  mode: 'production',
+  entry: './dist/cjs/jstp',
   output: {
-    path: path.join(__dirname, 'dist'),
-    filename: 'jstp.min.js',
-    sourceMapFilename: 'jstp.min.js.map',
-    libraryTarget: 'assign',
+    path: path.join(__dirname, 'dist', 'umd'),
+    filename: 'jstp.js',
+    sourceMapFilename: 'jstp.js.map',
+    libraryTarget: 'umd',
     library: ['api', 'jstp'],
   },
-  module: {
-    rules: [{
-      test: /\.js$/,
-      loader: 'babel-loader',
-      options: {
-        cacheDirectory: true,
-      },
-    }],
-  },
   devtool: 'source-map',
   bail: true,
   plugins: [
-    new webpack.optimize.UglifyJsPlugin({
-      sourceMap: true,
-    }),
     new webpack.BannerPlugin({
       banner: license,
     }),
   ],
+  node: {
+    net: false,
+    tls: false,
+    // TODO: support WebCrypto API in lib/common.js and uncomment this
+    // crypto: 'empty',
```

Comment:
This will both reduce the browser build size, like, three times, and also improve performance.